### ISSUE TITLE
Django template injection

### ISSUE THREAD

**Issue by sectroyer** [vladko312/SSTImap#15]
Looks SSTI is not able to detect Django template injection like in PortSwigger's 
[Server-side template injection with information disclosure via user-supplied objects](https://portswigger.net/web-security/server-side-template-injection/exploiting/lab-server-side-template-injection-with-information-disclosure-via-user-supplied-objects) Lab.

**Comment by vladko312** [comment]
Django template engine lacks exploitable execution capabilities, so exploiting it is different from other engines and focuses more on extracting variables. I might add support in the future.

**Comment by sectroyer** [comment]
Yes but detection would be nice that least know that "something is up" :) Also you can print an "info" that it's "worth to check" stuff like debug or secret key :) Usually that's enough to report the issue to the client

**Comment by vladko312** [comment]
@selectfromblackhydra Which version of Django would these payloads work on? They seem like payloads for Jinja2 used inside Django framework, not payloads for Django Template Engine. Also, the formatting seens broken.

**Comment by vladko312** [comment]
@selectfromblackhydra Can you check the value of `TEMPLATES` variable in `settings.py`? What does it say?

**Comment by vladko312** [comment]
@selectfromblackhydra This is just a file inside a Django project. What is in the settings file? And which payloads have you tested in your lab?
